### PR TITLE
Modify README

### PR DESCRIPTION
**Pull request by tiewweijian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+1. Change directory to user-service by running `cd question-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## Frontend Service
+Ensure that the URLs in frontend calls `localhost` instead of the deployed URL. 
+1. Change directory to frontend by running `cd frontend` 
+2. Ensure dependencies are downloaded locally `npm install`
+3. Start frontend by running `npm run start`
+4. The default browser will open with the application opened
```
Comment (zixin448) [inline README.md:59]: maybe can change to "...will open when the application starts running"

File: README.md [hunk 2]
```diff
@@ -1,14 +1,59 @@
...
-## Frontend
-1. Install npm packages using `npm i`.
-2. Run Frontend using `npm start`.
+Ensure that frontend directs to `localhost:8000` for user-service APIs. 
+
+1. Change directory to user-service by running `cd user-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## Collaboration Service
```
Comment (zixin448) [inline README.md:26]: Change the url for io connection to localhost:3000 to connect to frontend locally

File: README.md [hunk 3]
```diff
@@ -1,14 +1,59 @@
...
+3. Start the user-service by running `npm run start` 
+
+## History Service
+Ensure that frontend directs to `localhost:8005` for history-service APIs. 
+
+1. Change directory to user-service by running `cd history-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## Matching Service
```
Comment (zixin448) [inline README.md:40]: Change the url for io connection to localhost:3000 to connect to frontend locally

File: README.md [hunk 4]
```diff
@@ -1,14 +1,59 @@
...
+3. Start the user-service by running `npm run start` 
+
+## Question Service 
+Ensure that frontend directs to `localhost:8002` for question-service APIs. 
+
+1. Change directory to user-service by running `cd question-service`
+2. Ensure the dependencies are downloaded locally `npm install` 
+3. Start the user-service by running `npm run start` 
+
+## Frontend Service
```
Comment (zixin448) [inline README.md:54]: Run frontend locally on localhost:3000

File: README.md [hunk 5]
```diff
@@ -1,14 +1,59 @@
...
+Ensure that the URLs in frontend calls `localhost` instead of the deployed URL. 
+1. Change directory to frontend by running `cd frontend` 
+2. Ensure dependencies are downloaded locally `npm install`
```
Comment (likeabowx) [inline README.md:57]: This one need the ` --legacy-peer-deps` flag

File: README.md [hunk 6]
```diff
@@ -1,14 +1,59 @@
...
+# CS3219-AY22-23-S1-G13
 
-This is a template repository for CS3219 project.
+# Introduction 
+This project is an adaptation of the project PeerPrep that has been completed in the previous semesters of CS3219. This project provides a collaborative technical interview training platform, where one can receive a coding question, and collaborate with someone to answer the question. 
+
+
+# Running Project on Deployed Service
+
+The project is currently deployed [here](https://frontend-66acladbaq-uc.a.run.app). All APIs are also deployed, and able to be accessed through the frontend from the link.
```
Comment (likeabowx) [inline README.md:9]: Should we list out all the deployed urls of the different services here? Technically they can just try out the services in postman or something

Comment (tiewweijian) [reply README.md:9]: Perhaps we could put this in the report?

File: README.md (hunk 1 above)
Comment (tiewweijian) [reply README.md]: Resolved in new commit

File: README.md (hunk 2 above)
Comment (tiewweijian) [reply README.md:26]: Resolved in new commit

**Pull request merged**